prow.go

```diff
@@ -159,10 +159,11 @@ func (m *jobManager) stopJob(name, cluster string) error {
 		return err
 	}
 
-	// Deleting the prowjob pod will gracefully terminate template or step based jobs. In the case of
-	// steps, this includes running post steps.
-	klog.Infof("ProwJob pod for job %q will be deleted", name)
-	return m.clusterClients[cluster].CoreClient.CoreV1().Pods(m.prowNamespace).Delete(context.TODO(), name, metav1.DeleteOptions{})
+	klog.Infof("ProwJob pod for job %q will be aborted", name)
+	pj.SetComplete()
+	pj.Status.State = prowapiv1.AbortedState
+	_, err = m.prowClient.Namespace(m.prowNamespace).Update(context.TODO(), prow.ObjectToUnstructured(&pj), metav1.UpdateOptions{})
```

Comment:
Pass a real context?

Reply:
Replaced it with `Background` (not sure if its a great choice, other parts of the code just use `TODO`)

Reply:
Unfortunately, as @vrutkovs points out, there currently is not any relevant context configured anywhere in the code.  I would say it should probably remain as `context.TODO` and we can open a card to improve the context support of the chat-bot, as a whole, and not just for this one particular call.
@stevekuznetsov You placed the hold on this PR for the lack of a context, does this sound OK to you?

Reply:
I have created the following [ticket](https://issues.redhat.com/browse/DPCR-82)